architecture/core_concepts/routes.adoc

```diff
@@ -353,11 +353,11 @@ number of running servers changing, many clients will be
 directed to different servers. This algorithm is generally
 used with xref:routes-sticky-sessions[passthrough routes].
 
-The `*ROUTER_TCP_BALANCE_SCHEME*` environment variable sets the default
-for passthorugh routes.  The `*ROUTER_LOAD_BALANCE_ALGORITHM*` environment
-variable sets the default strategy for the router for the remaining routes.
+The `*ROUTER_TCP_BALANCE_SCHEME*` xref:env-variables[environment variable] sets the default
```

Comment:
Also, there *might* be an issue with tagged text right before a link where the space is not always picked up. One solution might be to put the tagged text into the link. 
xref:env-variables[`ROUTER_TCP_BALANCE_SCHEME` environment variable] 
But, I won't know until the next weekly publish on Monday or Tuesday next.